src/components/details/index.jsx

```diff
@@ -42,6 +42,14 @@ const ListItem = ({ icon, title, value, link, skeleton = false }) => {
   );
 };
 
+const isCompanyMention = (company) => {
+  return company.startsWith('@') && !company.includes(' ');
```

Comment:
I'm not fully sure what the exact rules are for a mention, I think a mention should start with a `@` and cannot contain spaces.

Reply:
Looks workable.